tests/tests_unit/rules/importer/test_ontology2rules.py

```diff
@@ -0,0 +1,12 @@
+from cognite.neat.rules._importer._owl2rules import OWLImporter
+from cognite.neat.rules.examples import wind_energy_ontology
+
+
+def test_owl2transformation_rules() -> None:
+    # Arrange
+    owl_importer = OWLImporter(wind_energy_ontology)
+    rules = owl_importer.to_rules(make_compliant=True)
+
+    # Assert
+    assert str(rules.metadata.namespace) == "https://kg.cognite.ai/wind/"
+    assert len(rules.classes.data) == 71

```

Comment:
The rules.classes is a user list, this is to access the underlying list. However, you can also just do
```suggestion
    assert len(rules.classes) == 71

```